lib/govuk_navigation_helpers.rb

```diff
@@ -1,5 +1,39 @@
 require "govuk_navigation_helpers/version"
 
 module GovukNavigationHelpers
-  # Your code goes here...
+  class NavigationHelper
+    def initialize(content_item)
+      @content_item = content_item
+    end
+
+    # Generate a breacrumb trail
+    #
+    # @return [Array<Hash>] Each item is a hash containing `:title` and `:url` for one link in the breadcrumb
+    def breadcrumbs
```

Comment:
Perhaps premature, but I would like it if this method would delegate the breadcrumb making to a `Breadcrumbs` class. That way we can test the breadcrumb & sidebar functionality seperately.